moveit_ros/moveit_servo/test/config/servo_settings_low_latency.yaml

```diff
@@ -55,7 +55,7 @@ command_out_topic: /panda_arm_controller/joint_trajectory # Publish outgoing com
 
 ## Collision checking for the entire robot body
 check_collisions: true # Check collisions?
-collision_check_rate: 5 # [Hz] Collision-checking can easily bog down a CPU if done too often.
+collision_check_rate: 5.0 # [Hz] Collision-checking can easily bog down a CPU if done too often.
```

Comment:
didn't andy make another pr that added all the decial places to this?

Reply:
> It seems that there are a ton of changes in this PR that are unrelated to this PR. Please refactor those into separate PRs or just remove them from this PR if they don't belong here.

Seems I did something wrong while squashing I will fix those today.